library-v3/build.gradle

```diff
@@ -56,7 +56,7 @@ dependencies {
     implementation 'com.google.android.libraries.maps:maps:3.1.0-beta'
     implementation 'com.android.volley:volley:1.2.1' // TODO - Remove this after Maps SDK v3 beta includes Volley versions on Maven Central
     implementation 'com.google.android.gms:play-services-basement:17.6.0'
-    implementation 'com.google.android.gms:play-services-base:17.2.1'
+    implementation 'com.google.android.gms:play-services-base:17.6.0'
```

Comment:
This came over from [this commit](https://github.com/googlemaps/android-maps-utils/pull/972/commits/751277ce62c503a296293694ce218194f114626f) on the [old master branch](https://github.com/googlemaps/android-maps-utils/tree/master). It's the only of the 5 commits ahead of the new main branch master had that haven't since been updated to newer dependencies already on main. Looks like the chore bot is now doing those dependency updates on main. Might be able to delete the master branch now entirely.

Reply:
Huh, strange - thanks for pointing that out. Not sure why dependabot didn't bump that in `main`.

@arriolac thoughts on removing `master` branch entirely? It might reduce confusion for folks that have it cloned locally on their machine who haven't realized that the main branch switched to `main`.

Reply:
++ to deleting `master` completely to avoid any confusion. Will go ahead and do that.